### ISSUE TITLE
Two of the three "sources" links are broken, and the third is just to a home page. The only publication evidence (under ameliorates) does not connect the drug to the disease.

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#161]

**Comment by capasfield** [comment]
From TAQA - 
There are issues with the source links that are listed under providence. Some have expired SSL certificates. Some Smart API links don't exist any more. 
The SRI reference link and the omnicorp link are broken. We need source URLs. 
This is a misparse. 

Ask Sierra to run a script to go through the URLs in the infores catalog to make sure they "go somewhere" (return a 200).

Does the SRI reference graph turn in SEMMEDDB?
Kevin - make sure references are accurate

**Comment by cbizon** [comment]
Looking at this further, the publication is coming from ctd originally, so it probably represents a curator making a mistake.

FWIW, though, watching everybody struggle with trying to figure out where the paper came from I made another issue here: 
![image](https://user-images.githubusercontent.com/10744350/235212571-b1af2247-97da-492d-bedc-847213ef49f5.png)

**Comment by kevinschaper** [comment]
Is there a way to trace how an edge coming in with SRI Ref KG as primary_knowledge_source got there? We don't use that in any of our knowledge_source fields. 

I think our 2023-03-17 release is the latest that has gone into translator. The knowledge sources are: 

| primary_knowledge_source |           aggregator_knowledge_source            |
|--------------------------|--------------------------------------------------|
| infores:xenbase          | infores:monarchinitiative                        |
| infores:hpoa             | infores:monarchinitiative                        |
| infores:string           | infores:monarchinitiative                        |
| infores:reactome         | infores:monarchinitiative                        |
| infores:bgee             | infores:monarchinitiative                        |
| infores:omim             | infores:monarchinitiative                        |
| infores:sgd              | infores:monarchinitiative|infores:alliancegenome |
| infores:wormbase         | infores:monarchinitiative|infores:alliancegenome |
| infores:flybase          | infores:monarchinitiative|infores:alliancegenome |
| infores:zfin             | infores:monarchinitiative|infores:alliancegenome |
| infores:mgi              | infores:monarchinitiative|infores:alliancegenome |
| infores:rgd              | infores:monarchinitiative|infores:alliancegenome |
| infores:panther          | infores:monarchinitiative                        |
| infores:chebi            | infores:phenio                                   |
| infores:emapa            | infores:phenio                                   |
| infores:fbbt             | infores:phenio                                   |
| infores:go               | infores:phenio                                   |
| infores:hp               | infores:phenio                                   |
| infores:upheno           | infores:phenio                                   |
| infores:mondo            | infores:phenio                                   |
| infores:mpath            | infores:phenio                                   |
| infores:mp               | infores:phenio                                   |
| infores:uberon           | infores:phenio                                   |
| infores:wbphenotype      | infores:phenio                                   |
| infores:wbbt             | infores:phenio                                   |
| infores:xpo              | infores:phenio                                   |
| infores:zfa              | infores:phenio                                   |
| infores:zp               | infores:phenio                                   |
| infores:zfin             | infores:monarchinitiative                        |
| infores:goa              | infores:monarchinitiative                        |

We also don't have any treats edges. Matt just gave me the smart-api link, so I'll see if I can do any other debugging to understand where this is coming from.

**Comment by cbizon** [comment]
@EvanDietzMorris is this happening when we pull it into plater?

**Comment by sierra-moxon** [comment]
Note that Biolink has a github action to prevent broken URLs to be added to the infores catalog and it checks existing URLs as well.

**Comment by bill-baumgartner** [comment]
@sierra-moxon, @capasfield  Just FYI - Kevin Cohen is on leave and is not currently participating in Translator, so I don't think he is the correct person to be assigned to this issue.

**Comment by sierra-moxon** [comment]
Retesting this morning on CI and I think this can be closed.  I waited for all the results to come back; I clicked through the top 5 chemicals and all the links in the Publications and Sources tabs returned valid pages.  Some results continue to only have improvingAgent as a source (and no publications), but the link for improvingAgent is updated in the infores catalog to the TranslatorWiki page associated with that ARA (and even the one that is there now works and provides some information about improving in the meantime).  

closing.
